dxr/filters.py

```diff
@@ -192,3 +192,44 @@ def highlight_content(self, result):
         return ((entity['start'], entity['end'])
                 for entity in result.get(self._needle, ()) if
                 self._should_be_highlit(entity))
+
+
+class QualifiedNameFilterBase(NameFilterBase):
```

Comment:
Yep, you'll find somebody else has already factored this up.  :-) You might want to merge master into your branch at some point.